### PR TITLE
Reco fix api

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Reco/Integrations/Reco/README.md [hunk 1]
```diff
@@ -379,8 +379,8 @@ Get files exposed to a specific email address
 | Reco.Assets.asset | Unknown | Json string of the asset's url and the name | 
 | Reco.Assets.data_category | String | The data category of the asset | 
 | Reco.Assets.data_categories | String | The data categories of the asset |
-| Reco.SensitiveAssets.location | String | The path of the asset. |
-| Reco.SensitiveAssets.source | String | SaaS tool source of the asset. |
```
Comment (jlevypaloalto) [inline Packs/Reco/Integrations/Reco/README.md:383]: Why are we removing those paths?

**Review by jlevypaloalto — COMMENTED** [review]
Hi @GalNakash-RecoLabs,
Thank you for your contribution!
Great work, just a short comment:

**Comment by GalNakash-RecoLabs** [comment]
Mai asked me to fix something from the latest PR

Sent via Superhuman iOS ***@***.***>

File: Packs/Reco/Integrations/Reco/README.md (hunk 1 above)
Comment (GalNakash-RecoLabs) [reply Packs/Reco/Integrations/Reco/README.md:383]: @maimorag  can you explain here

Comment (maimorag) [reply Packs/Reco/Integrations/Reco/README.md:383]: Yeah sure, in the last PR, there was a mismatch between the context path as stated in the yml compared to README, so know @GalNakash-RecoLabs fixed here the correct paths

**Comment by jlevypaloalto** [comment]
@GalNakash-RecoLabs, looks good!

Please merge from master. Note that the location of the `git_pull_master_into_fork.sh` file has changed to `./.circleci/git_pull_master_into_fork.sh`.

**Comment by GalNakash-RecoLabs** [comment]
Done

**Comment by jlevypaloalto** [comment]
@GalNakash-RecoLabs, looks great!

Please merge from master. Note that the location of the `git_pull_master_into_fork.sh` script has moved to `./.circleci/git_pull_master_into_fork.sh`.

**Comment by GalNakash-RecoLabs** [comment]
Done


Sent via Superhuman ***@***.***>

**Review by jlevypaloalto — APPROVED** [review]
Looks great!

**Pull request merged**